yaptide/utils/s3_uploader.py

```diff
@@ -0,0 +1,81 @@
+import boto3
+import os
+from cryptography.fernet import Fernet
+import sys
+import tempfile
+from botocore.exceptions import NoCredentialsError, EndpointConnectionError, ClientError
+from pathlib import Path
+
+
+def encrypt_file(file_path: Path, key: str) -> Path:
+    """Encrypts a file using Fernet"""
+    if not file_path.is_file():
+        raise ValueError("Invalid file path or file does not exist.")
+    with open(file_path, "rb") as file:
+        original = file.read()
+    fernet = Fernet(key)
+    encrypted = fernet.encrypt(original)
```

Comment:
I'd rather create a function which encrypts files and gives back on return a string with file contents. Then file saving and deleting can be easily handled by callers